### PR TITLE
Include changelog for sampling and optimisation initialisation PR

### PR DESCRIPTION
**Pull request by ben18785**
Includes missing changelog for #1322 

Also moved documentation to utilities folder.

### CODE REVIEW

**Comment by MichaelClerx** [comment]
Not necessarily a stochastic function?

**Comment by MichaelClerx** [comment]
One more thing, could we move the docs for `sample_initial_points` out of "Running an MCMC routine" ?
E.g. to "Utilities" or "Core classes and methods" ?

**Comment by ben18785** [comment]
> Not necessarily a stochastic function?

So, there are no checks that the function supplied is stochastic. But I'm not sure why you'd use it if it weren't?

**Comment by ben18785** [comment]
But the function used to sample from them is....

**Comment by MichaelClerx** [comment]
To me that's sampling from the boundaries plus a check that `P(stochastic function) != 0`, not sampling from P?

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ All notable changes to this project will be documented in this file.
 
 ### Added
 - [#1383](https://github.com/pints-team/pints/pull/1383) Added a method `toy.TwistedGaussianDistribution.untwist` that turns samples from this distribution into samples from a multivariate Gaussian.
+- [#1322](https://github.com/pints-team/pints/pull/1322) Added a method `sample_initial_points` that allows users to generate random initial points with finite metrics (either log-probabilities or error measures) and, if specified, ensures that the points are within the boundaries. The randomness in the initial points can be determined by a user-supplied stochastic function. When a stochastic function is not supplied, if the metric is a log-posterior, the prior will automatically be sampled from; finally, if a stochastic function is not supplied and if the metric is not a log-posterior, if boundaries are supplied, then initial points will automatically be generated from within these.
```
Comment (MichaelClerx) [inline CHANGELOG.md:9]:
```suggestion
- [#1322](https://github.com/pints-team/pints/pull/1322) Added a method `sample_initial_points` that allows users to generate random points with finite metrics (either log-probabilities or error measures) to use as starting points for sampling or optimisation.
```

**Review by MichaelClerx — APPROVED** [review]

**Pull request merged**